### PR TITLE
Add first version of Machine Shutdown script

### PR DESCRIPTION
**Pull request by anacso17**
Hi guys, this branch was tested during machine study in August 29th. The idea is to migrate this script to siriuspy, but for now we will keep this script.

@xresende , @CarlosvieiraHB , we started our work in a branch called add-machine-shutdown. Because of a problem with a git rebase command, I polluted the history of the branch. Fortunately, I have saved a branch with the original history, and used this for the PR. That's because the PR is from this new branch add-machine-shutdown-script. Sorry for the mess.

### CODE REVIEW

File: bin/sirius-script-procedures-machine-shutdown.py
Comment (fernandohds564) [inline bin/sirius-script-procedures-machine-shutdown.py:58]: Remember to change this name once lnls-sirius/dev-packages#1016 is merged.

File: bin/sirius-script-procedures-machine-shutdown.py
Comment (fernandohds564) [inline bin/sirius-script-procedures-machine-shutdown.py:78]: Same here

File: bin/sirius-script-procedures-machine-shutdown.py
Comment (fernandohds564) [inline bin/sirius-script-procedures-machine-shutdown.py:185]: Here too

File: bin/sirius-script-procedures-machine-shutdown.py
Comment (fernandohds564) [inline bin/sirius-script-procedures-machine-shutdown.py:198]: Also here. Remember that the interface of the constructor changed.

File: bin/sirius-script-procedures-machine-shutdown.py
Comment (fernandohds564) [inline bin/sirius-script-procedures-machine-shutdown.py:599]: Are you sure we want this interactive behavior? As far as I understood, this script should be configured to run automatically in a pre-determined time.

Comment (fernandohds564) [reply bin/sirius-script-procedures-machine-shutdown.py:599]: I think we could at least put a timeout waiting for the response and continue otherwise.

Comment (anacso17) [reply bin/sirius-script-procedures-machine-shutdown.py:599]: Hi @fernandohds564 ! This is only for the first tests. We also have to define which steps will block the procedure.

**Review by xresende — APPROVED** [review]

File: bin/sirius-script-procedures-machine-shutdown.py
Comment (fernandohds564) [reply bin/sirius-script-procedures-machine-shutdown.py:620]: but we could at least put a timeout here, couldn't we?
In case you decide for the timeout, the following code works:
```python3
import select
import sys

def input_(msg, timeout=5):
    print(msg, end='')
    sys.stdout.flush()
    i, o, e = select.select([sys.stdin], [], [], 5)
    if i:
        return sys.stdin.readline().strip()
    else:
        print('timeout')
    return

out = input_('Type anything and press Enter (you have 5s to do so): ', 5) 
```

File: bin/sirius-script-procedures-machine-shutdown.py
Comment (fernandohds564) [inline bin/sirius-script-procedures-machine-shutdown.py:678]: I think we could make use of the the new `Device` interface and only connect with the PVs that will be used here, what do you think? I think it will be worth the effort, at least for the large devices.

File: bin/sirius-script-procedures-machine-shutdown.py
Comment (anacso17) [reply bin/sirius-script-procedures-machine-shutdown.py:335]: Good catch, done!

File: bin/sirius-script-procedures-machine-shutdown.py
Comment (anacso17) [reply bin/sirius-script-procedures-machine-shutdown.py:406]: the method `set_voltage_incrate` already checks the readback value

File: bin/sirius-script-procedures-machine-shutdown.py
Comment (anacso17) [reply bin/sirius-script-procedures-machine-shutdown.py:685]: Agree!! I will do it as soon as possible!

File: bin/sirius-script-procedures-machine-shutdown.py
Comment (anacso17) [reply bin/sirius-script-procedures-machine-shutdown.py]: This value (62) was an input from GOP/RF

File: bin/sirius-script-procedures-machine-shutdown.py
Comment (anacso17) [reply bin/sirius-script-procedures-machine-shutdown.py:634]: We will replace this with the following commented lines, that checks the new PV with the remaining time for tunnel access, as soon as the interlock IOC is ready

File: bin/sirius-script-procedures-machine-shutdown.py
Comment (anacso17) [reply bin/sirius-script-procedures-machine-shutdown.py:692]: done!

File: bin/sirius-script-procedures-machine-shutdown.py
Comment (anacso17) [reply bin/sirius-script-procedures-machine-shutdown.py]: > Wouldn't be better to define a class constant here?

done!

Comment (anacso17) [reply bin/sirius-script-procedures-machine-shutdown.py]: > I think we could implement a this turnoff (and also a turn on) procedure in that device.

maybe we could migrate to there, but it could be a future enhancement (I won't be able to work on this before october)

File: bin/sirius-script-procedures-machine-shutdown.py
Comment (anacso17) [reply bin/sirius-script-procedures-machine-shutdown.py]: ...done!

**Review by fernandohds564 — APPROVED** [review]

**Review by fernandohds564 — APPROVED** [review]

**Pull request merged**
